### DISCUSSION TITLE
lattice distortion correction in IGOR

### DISCUSSION THREAD

**Discussion by daweihuhu** [yuksk/SIDAM#69] [Q&A]
Sorry to bother you. I was wondering how you perform lattice distortion correction in Igor pro. We all know that the topo and mapping data obtained from STM can differ somewhat from the actual lattice structure. How do you address this issue in IGOR? Thank you!

**Comment by yuksk** [comment]
I have my own code to perform the correction. (At this time, I do not plan to make it public.)
